handlers/line/handler.go

```diff
@@ -317,6 +317,8 @@ func (h *handler) Send(ctx context.Context, msg courier.MsgOut, res *courier.Sen
 
 		if err == nil {
 			jsonMsgs = append(jsonMsgs, string(jsonMsg))
+		} else {
+			clog.Error(courier.NewChannelError("", "", err.Error()))
```

Comment:
This line will help see why the attachments was not send

Reply:
if marshaling those things is creating an error.. .then just return that err (`return err`) and we'll fix it... but I'm not convinced this is where the problem is